app/auth/callback/unilogin/route.ts

```diff
@@ -91,12 +93,15 @@ export async function GET(request: NextRequest) {
     }
 
     // Set user info.
+    // TODO: After Publizon allows DDF test users to loan, we can remove thie if statement.
     session.userInfo = {
       sub: userinfo.sub,
       uniid: introspect.uniid,
-      institution_ids: introspect.institution_ids,
+      // TODO: Rename this into institutionIds
+      institution_ids:
+        institutionId === "A04441" ? ["101047"] : getInstitutionIds(introspect.institution_ids),
```

Comment:
Please write why we are doing this 😅

Reply:
It is explained right above on line 96. Do you think we need more?

Reply:
Sorry you're tight. Move along, nothing to see here.